mautic/mautic
# php 8.0 support

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Minimal changes to support PHP 8.0 for Mautic 4 (therefore it's based on 4.3).

This is the successor of #9969 (and makes #11048 obsolete).

To support PHP 8.1,  more (breaking) changes are needed, as dependencies (of dependencies) only support PHP 8.1 in a version with BC breaking changes.



#### Steps to test this PR:

* check the outcome of the test
* go through Mautic with error reporting on and reporting all errors



<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11217"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
